src/PowerShellEditorServices/Console/UnixConsoleOperations.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using UnixConsoleEcho;
+
+namespace Microsoft.PowerShell.EditorServices.Console
+{
+    internal class UnixConsoleOperations : IConsoleOperations
```

Comment:
Does this apply to Unix and Linux?

Reply:
Yeah, I'm using "Unix" to refer to Mac and Linux.  Not sure if there's a better term, but that's what corefx does.